app/javascript/App/bundles/ExchangeListings/components/Tips.jsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react'
+import Icon from '../../../components/Icon'
+import chartIcon from '../../../images/chartIcon.svg'
+import filterIcon from '../../../images/filterIcon.svg'
+import listIcon from '../../../images/listIcon.svg'
+
+const titleStyle = {
+  fontSize: '1.13rem',
+  fontWeight: 'bold',
+}
+
+titleStyle.paddingLeft = 15
+titleStyle.paddingBottom = 10
```

Comment:
Why it's moved out of object definition?

Reply:
At one point we needed these properties for responsive design, so this was moved out.